src/lib/registry/severity-registry.js

```diff
@@ -0,0 +1,47 @@
+const Severity = require('../severity');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
const Severity = require("../severity");
```